chain/blockvalidator.go

```diff
@@ -388,11 +392,6 @@ func HeaderCheck(b *block.Block) error {
 		return fmt.Errorf("signer check failed: %v", err)
 	}
 
-	err = SignatureCheck(header)
```

Comment:
This will skip signature check for ALL header check call and proposal signature is no longer checked. Instead of removing it, add a `IsSignatureChecked` field in header, just like `IsHeaderChecked` field in block.

Reply:
I moved this check to the `SaveBlock` function